pkg/cmd/server/api/v1/conversions.go

```diff
@@ -131,6 +131,10 @@ func addDefaultingFuncs(scheme *runtime.Scheme) {
 			if len(obj.BindNetwork) == 0 {
 				obj.BindNetwork = "tcp4"
 			}
+			if obj.AllowRecursiveQueries == nil {
```

Comment:
Wait, so our default will soon be to allow recursive lookups unless the config files override this?